harshita/level1/discount.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    double quantity,cost=100,amount=0;
+    cout << "Enter Quantity : ";
+    cin >> quantity ;
+    if(quantity > 1000){
```

Comment:
Here we have to check is costs less than 1000 or not !! so it will `if(quantity*100 > 1000)`